css/index.css

```diff
@@ -1,3 +1,476 @@
+ 
+  
+  /* cta */
+
+  @media(max-width: 800px){
+      .cta {
+        display: flex;
+        justify-content: center;
+        align-content: space-around;
+      }
+  }
+  
+  .cta .cta-text {
+	display: inline-block;
+	vertical-align: top;
+	font-size: 77px;
+	margin-top: 120px;
+	text-align: center;
+	padding: 0px 75px;
+  }
+  
+  .cta .cta-text button {
+    background: white;
+    color: black;
+	vertical-align: top;
+	margin-top: 6px;
+	padding: 4px 44px;
+    font-size: 17px;
+    border-radius: 2px;
+	border: 1px solid black;
+	cursor: pointer;
+  }
+  
+  .cta .cta-text button:hover {
+	background: black;
+	color: white;
+  }
+  
+  .cta img {
+	display: inline-block;
+	margin-top: 69px;
+  }
+
+  @media(max-width: 800px){
+    .cta img {
+        display: none;
+    }
+  }
+  
+  .main-content {
+	border-top: 1px solid black;
+	margin-top: 76px;
+	padding-top: 39px;
+  }
+  
+  .main-content .top-content {
+	padding: 0 10px;
+  }
+  @media(max-width: 800px){
+     .main-content .top-content{
+         display: flex;        
+     }
+  }
+  @media(max-width: 500px){
+    .main-content .top-content{
+      flex-direction: column;
+      padding-left: 0;
+    }
+  }
+ 
+  
+  .main-content .top-content .text-content {
+	width: 410px;
+	display: inline-block;
+	margin-right: 34px;
+  }
+  @media(max-width: 500px){ 
+    .main-content .top-content .text-content{ 
+        width: 100%; 
+    } 
+  }
+  
+  .main-content .top-content .text-content:last-child {
+	margin-right: 0;
+  }
+  @media(max-width: 500px){
+    .main-content .top-content .text-content:last-child {
+        margin: 30px 0 0 0;
+    }
+  }
+  
+  .main-content .middle-img {
+	margin-top: 45px;
+  }
+  @media(max-width: 800px){
+    .main-content .middle-img{
+        max-width: 100%;
+        height: auto;
+    }
+  }
+  @media(max-width: 500px){
+      .main-content .middle-img{
+          height: auto;
+      }
+  }
+  
+  .main-content .bottom-content {
+	margin-top: 37px;
+	padding: 0 10px;
+  }
+  @media(max-width: 800px){
+    .main-content .bottom-content{
+        display: flex;
+    }
+  }
+  @media(max-width: 500px){
+    .main-content .bottom-content{
+        flex-direction: column;
+        align-items: stretch;
+        justify-content: space-between;
+        padding: 0;
+        height: 420px;
+    }
+  }
+ 
+  
+  .main-content .bottom-content .text-content {
+	width: 275px;
+	display: inline-block;
+	margin-right: 12px;
+  }
+   @media(max-width: 500px){ 
+     .main-content .bottom-content .text-content{ 
+         width: 100%; 
+     } 
+  } 
+ 
+  
+  .main-content .bottom-content .text-content:last-child {
+	margin-right: 0;
+  }
+  
+  .contact {
+	margin-top: 57px;
+	border-top: 1px solid black;
+	padding: 39px 10px 0;
+  }
+  
+  .contact p {
+	margin: 20px 0;
+  }
+  
+  footer {
+	margin: 32px 0;
+	text-align: center;
+	font-size: 14px;
+  }
+
+  
+
+  /* services */
+
+ .services .services-header-img {
+	  display: block;
+	margin: 30px 0;
```

Comment:
You have some inconsistent spacing; I'd try using the prettier extension (https://prettier.io/) to remedy that. You can format your code with a press of a few keys.